tasks/preprocess_upload.py

```diff
@@ -70,7 +76,12 @@ def run_impl(
                     lock_name=lock_name,
                 ),
             )
-            return {"preprocessed_upload": False}
+            return {"preprocessed_upload": False, "reason": "unable_to_acquire_lock"}
```

Comment:
I would also maybe add the check to see if the lock is taken here in order to decided whether to log.warning("unable to acquire lock") or log.info("PreProcess task is already running"), because it's possible that the `LockError` is caused by another task having already taken it